### DISCUSSION TITLE
[V3] Need more info and examples for attributes in the docs

### DISCUSSION THREAD

**Discussion by ev3rlost** [livewire/livewire#5981] [3. Feature ideas]
I would love to see a seperate docs page on atrributes itself, and a few practical examples. Like, what are the Livewire out-of-the-box ones and how to write your own. Caleb demoed the `#[Format('d/m/Y')]` (or was it `/** @format d/m/Y */`) attribute on Laracon '22 but I have no idea how to write it on my own.

**A searchbar in the docs would be MORE than welcome.** Maybe even a seperate page with all the php artisan livewire commands and what they do (I know just writing that in the console gives you all the info, but it should be in the docs imho).

**Comment by joshhanley** [comment]
@ev3rlost thanks for the suggestion! PRs to update the docs are welcome! In regards to the search bar, that is coming soon 🙂
